SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,55 +1,70 @@
 
-There are two kinds of Singleton Actors:
+You define a Singleton Actor by tagging an Actor with the `SpatialType=Singleton` class attribute. For example, if you are implementing a scoreboard, you probably want only one scoreboard in your world, so you can tag the scoreboard Actor as a Singleton Actor.
 
-* **Public Singleton Actors** - Singleton Actors which are replicated to [server-workers and client-workers]({{urlRoot}}/content/glossary#workers). [AGameState](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Public Singleton Actor.
-* **Private Singleton Actors** - Singleton Actors which are replicated to [server-workers]({{urlRoot}}/content/glossary#workers), but not accessible to [client-workers]({{urlRoot}}/content/glossary#workers). [AGameMode](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
+There are two types of Singleton Actors:
 
-You can define any class as a Singleton Actor. Unreal engine classes we have explicitaly tagged as Singleton Actors are -
+* **Public Singleton Actors** - Singleton Actors that are replicated to [server-workers and client-worker instances]({{urlRoot}}/content/glossary#workers). [`AGameState`](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) and `ALevelScriptActor` are Public Singleton Actors.
+* **Private Singleton Actors** - Singleton Actors that are replicated to [server-worker instances]({{urlRoot}}/content/glossary#workers), but not accessible to [client-worker instances]({{urlRoot}}/content/glossary#workers). [`AGameMode`](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
 
-1. AGameModeBase
-1. AGameStateBase
+You can define any class as a Singleton Actor. However, the following Unreal Engine classes are automatically tagged in source code as Singleton Actors, so you don’t need to define them:
 
-* As `SpatialType` is inheritable, all classes that derive off these classes are also considered Singleton Actors. You can opt out using the `NotSpatialType` tag.
+* `AGameModeBase`
+* `AGameStateBase`
+* `ALevelScriptActor`
 
-Each server-worker should instantiate their own local version of each Singleton Actor. For `AGameMode` and `AGameState`, Unreal Engine does this automatically. Client-workers receive Public Singletons Actors from the server-workers via the normal Actor replication lifecycle.
+> **Note**: Because the `SpatialType` class attribute is inheritable, all the classes that derive from `AGameModeBase`,  `AGameStateBase`, or `ALevelScriptActor` class are also Singleton Actors, which means that you don’t need to explicitly tag them in your code. You can opt out of this by tagging these classes with the [`NotSpatialType`]({{urlRoot}}/content/spatial-type#spatial-type) tag.
```

Comment:
```suggestion
> **Note**: Because the [`SpatialType`]({{urlRoot}}/content/spatial-type) class attribute is inheritable, all the classes that derive from `AGameModeBase`, `AGameStateBase`, or `ALevelScriptActor` are also Singleton Actors, which means that you don’t need to explicitly tag them in your code. You can opt out of this by tagging these classes with the [`NotSpatialType`]({{urlRoot}}/content/spatial-type#spatial-type) tag.
```